db/migrate/20210305213649_validate_add_null_constraint_to_sp_friendly_name.rb

```diff
@@ -0,0 +1,24 @@
+class ValidateAddNullConstraintToSpFriendlyName < ActiveRecord::Migration[6.1]
+  def up
+    safety_assured do
+      execute 'UPDATE service_providers SET friendly_name = issuer WHERE friendly_name IS NULL'
+
+      # following guidance from strong_migration
+      execute 'ALTER TABLE "service_providers" VALIDATE CONSTRAINT "service_providers_friendly_name_null"'
+    end
+
+    change_column_null :service_providers, :friendly_name, false
```

Comment:
Note that wrapping this in the `safety_assured` block does allow CI to pass.

Reply:
Verified that using a later version of PG does not throw this error ([CircleCI build](https://app.circleci.com/pipelines/github/18F/identity-idp/30062/workflows/4104cfe0-cc6d-4f10-b9a9-378b913b990f/jobs/52393))

Reply:
👍🏼 👍🏼 

Would we want to sneak in the CircleCI postgres version upgrade to 9.6 as part of this?

Reply:
Are we using 9.6 in prod?

Reply:
yep

Reply:
Yeah then I'll definitely update it. Thanks!